index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+   
+    <div id= "main">
+     <section id="title">
+       <h1 class = "h1">Dr. Norman Borlaug</h1>
+       <p  class = "p">The man who saved a billion lives</p>
+     </section>
+     <section id="img-div">
+       <img id="image" src="image/tribute-page-main-image.jpeg">
```

Comment:
Kindly add an alt attribute to specify an alternate text for an image

Reply:
- [ ] **[OPTIONAL]**
- Great adding the changes however, I think you could improve more on elaborating more on the changes your propose.
- Your reason here is fair enough but not really explicit. Kindly work on this.